canonical-web-and-design/ubuntu.com
# Updated sitemap

## Done

- Updated the sitemap.xml

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/sitemap.xml
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- paste the sitemap xml in a tool like https://xmlgrid.net/sitemapView.html to validay
